### ISSUE TITLE
[Bug]: The Idiomatic Text element does not italicize captions

### ISSUE THREAD

**Issue by An-anonymous-coder** [FreeTubeApp/FreeTube#6238]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

1. Open a video that has captions that are italicized using the Idiomatic Text element
2. Enable captions for that language

The captions will not be italicized, but will instead be surrounded by the Idiomatic Text element (`<i>...</i>`)

### Expected Behavior

The captions should be italicized or the Idiomatic Text element should be removed

### Issue Labels

accessibility issue, text/string issue

### FreeTube Version

v0.22.0-nightly-5178 Beta, 2b91b4b

### Operating System Version

Debian GNU/Linux 12 (bookworm) x86_64 6.1.0-28-amd64

### Installation Method

.deb

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

N/A

### Additional Information

Learn more about the Idiomatic Text element [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i)

The `English - English` captions for [this video](https://youtu.be/kJQP7kiw5Fk) use the Idiomatic Text element

### Nightly Build

- [X] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by absidue** [comment]
FreeTube is actually doing the right thing here.

If you look at the actual subtitle files you'll notice that the <i> are actually escaped so that they should appear as text instead of getting interpreted (yes this is likely a mistake from the video uploader for formatting the subtitles incorrectly). The fact that it is shown with italic text on YouTube's own website is likely a bug on their end with them not respecting that <i> should appear as text based on the subtitle files, if you view the same video in YouTube's official mobile apps, you'll notice that they show the <i> too.

**Comment by absidue** [comment]
My perspective on this is that we shouldn't intentionally break FreeTube, just because uploader's didn't setup their videos correctly, especially because it could break the actually correct subtitles on other videos.

Additionally FreeTube is doing the same thing as the official YouTube mobile apps, so the fact that YouTube's website is enabling the uploader to provide incorrectly formatted subtitles is likely unintentional and likely to be a bug in their website.
